runtime/doc/autocmd.txt

```diff
@@ -977,6 +979,15 @@ InsertLeavePre			Just before leaving Insert mode.  Also when
 							*InsertLeave*
 InsertLeave			Just after leaving Insert mode.  Also when
 				using CTRL-O |i_CTRL-O|.  But not for |i_CTRL-C|.
+							*KeyInputPre*
+KeyInputPre			When a key is typed. The pattern is
```

Comment:
I don't think "typed" is an accurate description of when this event is triggered. This is triggered before a key is processed.

Reply:
Fixed.